### ISSUE TITLE
BioBert 

### ISSUE THREAD

**Issue by kellyquek** [marcelbra/DocTagger#1]
Just to check. Would it be possible to use BioBert directly to obtain the 4 entities type - gene, drug, disease, species? The labels i obtained now are B-MISC etc..

**Comment by marcelbra** [comment]
Depends on what you mean with „directly“. You definitely need to fine-tune, as BioBert pre-trained is basically just a language model. If you do finetune, then yes, that’s possibly. But you need a dataset annotated with these 4 tags and need to change the tags list and tag_map in pred_ner.py accordingly.
